import-export-cli/cmd/deprecated/addApi.go

```diff
@@ -219,16 +219,16 @@ func createAPI(configMapNames []string, timestamp string, balInterceptors []stri
 		k8sOperation = k8sUtils.K8sApply
 		k8sSaveConfig = false
 	}
-	if len(balInterceptors) > 0 {
-		// set bal interceptors configmap name in API cr
-		apiCrd.Spec.Definition.Interceptors.Ballerina = balInterceptors
-	}
-	if len(javaInterceptors) > 0 {
-		//set java interceptors configmaps names in API cr
-		apiCrd.Spec.Definition.Interceptors.Java = javaInterceptors
-	} else {
-		apiCrd.Spec.Definition.Interceptors.Java = []string{}
-	}
+	//if len(balInterceptors) > 0 {
+	//	// set bal interceptors configmap name in API cr
+	//	apiCrd.Spec.Definition.Interceptors.Ballerina = balInterceptors
+	//}
+	//if len(javaInterceptors) > 0 {
+	//	//set java interceptors configmaps names in API cr
+	//	apiCrd.Spec.Definition.Interceptors.Java = javaInterceptors
+	//} else {
+	//	apiCrd.Spec.Definition.Interceptors.Java = []string{}
+	//}
```

Comment:
Here as well. The commented code?

Reply:
Fixed with e13921e5fc8c8e68de28237ef3e0fcaef1a51f9b